scripts/spo-apply-column-format/README.md

```diff
@@ -90,6 +89,77 @@ $fieldToFormat.UpdateAndPushChanges($true)
 > [!Note]
 > In 3rd method i use **$fieldToFormat.UpdateAndPushChanges($true)** but it is perfectly fine to use **Set-PnPField** command as in method 1 and 2
 
+# [CLI for Microsoft 365](#tab/cli-m365-ps)
+```powershell
+#site collection url
+$url="https://thejasey.sharepoint.com/sites/Examplesite"
+
+#list to be exported  
+$listName="Test"
```

Comment:
lets just add space next to `=` to increase readability and make it consistent with rest of the code 👍
`$url = "https://thejasey.sharepoint.com/sites/Examplesite"`
and
`$listName = "Test"`